onnxruntime/core/platform/windows/logging/etw_sink.cc

```diff
@@ -142,7 +142,7 @@ void EtwSink::SendImpl(const Timestamp& timestamp, const std::string& logger_id,
         TRACE_LOG_WRITE(TRACE_LEVEL_CRITICAL);
         break;
       default:
-        throw std::logic_error("Unexpected Severity of " + static_cast<int>(severity));
+        ORT_LOGIC_ERROR("Unexpected Severity of " + static_cast<int>(severity));
     }
```

Comment:
Should be able to just use ORT_THROW here.

Reply:
There are some logging test require catch logic_error exceptions, since we have the macro, will keep use it


---
In reply to: [476242828](https://github.com/microsoft/onnxruntime/pull/4894#discussion_r476242828) [](ancestors = 476242828)